neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -178,7 +181,7 @@ def create_experiment(self,
         # Initialize new experiment
         self._execute_alpha_operation(
             experiment=experiment,
-            operations=self._get_init_experiment_operations(entrypoint, params, tags),
+            operations=self._get_init_experiment_operations(name, entrypoint, params, tags),
```

Comment:
There are more arguments that should be supported (e.g. description and hostname). I guess it is a good moment to create an epic in our Jira and add tasks for such things so we will not forget about it.